test/evergreen.yml

```diff
@@ -154,6 +154,9 @@ functions:
         set -o verbose
 
         ${test_env_vars|} ${python_binary|python} ../test/suite/run.py ${unit_test_args|-v 2} ${smp_command|} 2>&1
+        # Dump stderr/stdout contents generated by the C libraries onto console
+        find "WT_TEST" -name "std*.txt" ! -empty -printf "\nContents from '%p':\n\n" -exec cat {} \;
+
```

Comment:
Actually this section of changes is suppose to be substituted by the `"dump stderr/stdout"` function call at the `post` section. I will get it removed.